Objects/complexobject.c

```diff
@@ -1339,16 +1343,16 @@ static PyMemberDef complex_members[] = {
 };
 
 static PyNumberMethods complex_as_number = {
-    (binaryfunc)complex_add,                    /* nb_add */
-    (binaryfunc)complex_sub,                    /* nb_subtract */
-    (binaryfunc)complex_mul,                    /* nb_multiply */
+    complex_add,                                /* nb_add */
```

Comment:
This structure modified a lot.  Maybe it's an opportunity to change it to include only added entries in style:
```c
static PyNumberMethods complex_as_number = {
    .nb_add = complex_add,
    .nb_subtract = complex_sub,
    .nb_multiply = complex_mul,
    .nb_power = complex_pow,
    [...]
};
```
```

Reply:
I wanted to do that as per https://github.com/python/cpython/issues/127679 but since Serhiy wanted a discussion first (https://github.com/python/cpython/issues/127679#issuecomment-2523046765) and since others wanted PEP-7 to be updated.

Reply:
I see not discussion opened so far.

Reply:
Oh.. then.. maybe I can do the change? well.. I actually did it for internal structures, but for exported ones, I don't think I did it in any of the other PR. I'll ask on the issue.

Reply:
> I can do the change?

Well, discussion is not started so far.  That's all:)